packages/components/src/CollapsiblePanel/CollapsiblePanel.component.js

```diff
@@ -47,18 +47,11 @@ function renderHeaderItem({ displayMode, className, ...headerItem }, key) {
 	switch (displayMode) {
 		case TYPE_STATUS: {
 			const { actions, ...restStatus } = headerItem;
-			const adaptActions = actions.map(
-				action => ({
-					...action,
-					onClick: getActionHandler(action.onClick, headerItem),
-				})
-			);
-			return (<Status
-				key={key}
-				actions={adaptActions}
-				{...restStatus}
-				className={css[className]}
-			/>);
+			const adaptActions = actions.map(action => ({
+				...action,
+				onClick: getActionHandler(action.onClick, headerItem),
+			}));
```

Comment:
`adapt` is meaningless. It doesn't describe the transformation applied to the actions. Please find a more descriptive name.

Reply:
done